src/mpid/ch4/netmod/ofi/ofi_send.h

```diff
@@ -213,11 +216,18 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_send_normal(const void *buf, MPI_Aint cou
     }
 
     send_buf = MPIR_get_contig_ptr(buf, dt_true_lb);
-    MPL_pointer_attr_t attr;
-    MPIR_GPU_query_pointer_attr(send_buf, &attr);
+
+    if (MPIDI_OFI_ENABLE_HMEM && data_sz >= MPIR_CVAR_CH4_OFI_GPU_RDMA_THRESHOLD) {
+        if (MPIDI_OFI_ENABLE_MR_HMEM) {
```

Comment:
Merge the conditions? Is there a place that `MPIDI_OFI_ENABLE_HMEM` and `MPIDI_OFI_ENABLE_MR_HMEM` is not checked together?

Reply:
Done